keras_nlp/models/distilbert/distilbert_models.py

```diff
@@ -0,0 +1,307 @@
+# Copyright 2022 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""DistilBERT backbone models."""
+
+import os
+
+import tensorflow as tf
+from tensorflow import keras
+
+from keras_nlp.layers.token_and_position_embedding import (
+    TokenAndPositionEmbedding,
+)
+from keras_nlp.layers.transformer_encoder import TransformerEncoder
+from keras_nlp.models.distilbert.distilbert_checkpoints import checkpoints
+from keras_nlp.models.distilbert.distilbert_checkpoints import (
+    compatible_checkpoints,
+)
+from keras_nlp.models.distilbert.distilbert_checkpoints import vocabularies
+
+
+def distilbert_kernel_initializer(stddev=0.02):
+    return keras.initializers.TruncatedNormal(stddev=stddev)
+
+
+def _handle_pretrained_model_arguments(
```

Comment:
I would actually go ahead and removes these for now (and the whole distilbert_checkpoints file). And we can add them in when we start actually providing the weights.

I think this is a good practice to get into, as we could basically cut a release whenever without the weights, and just be releasing the graph code and preprocessing code. (and still follow up with weights later)